js/features/mpc-stripes.js

```
@@ -0,0 +1,757 @@
+  } catch (err) {
+    return entry.handle.getFile();
+  }
+}
+
+// Same, but ensures the backup exists first — used by processing so the
+// original bytes are safe before we overwrite the file in place.
+async function backupAndReadCleanFile(entry, originalsDir) {
+  try {
+    const backup = await originalsDir.getFileHandle(entry.name);
+    return await backup.getFile();
+  } catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(no-unused-vars)